examples/advanced/border0_connector.tf

```diff
@@ -0,0 +1,142 @@
+// ===========================================================================
+// Create a connector with plugins
+// ===========================================================================
+
+resource "border0_connector" "my_connector" {
+  name = "my-connector"
+  description = "my connector"
+
+  aws_ec2_plugin {
```

Comment:
Plugins should not be prt of the connector config as they are standalone entities. 

This plugin also should be called aws_ec2_discovery_plugin -- as we may one day support non-discovery plugins